WikiClientLibrary/Generators/Fileusage.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using WikiClientLibrary.Generators.Primitive;
+using WikiClientLibrary.Infrastructures;
+using WikiClientLibrary.Sites;
+
+namespace WikiClientLibrary.Generators
+{
+    /// <summary>
+    /// Generates all the pages that transclude the specified file.
+    /// </summary>
+    /// <seealso cref="BacklinksGenerator"/>
+    /// <seealso cref="TranscludedInGenerator"/>
+    /// <seealso cref="TransclusionsGenerator"/>
+    public class FileusageGenerator : WikiPageGenerator
```

Comment:
Please use Pascal case here (FileUsageGenerator).

Reply:
I changed the class name and also fixed the file name (Generator was missing).